src/lib.rs

```diff
@@ -1503,3 +1516,7 @@ impl<T: Counter, F: Counter> PartialEq<Histogram<F>> for Histogram<T>
 #[path = "tests/tests.rs"]
 #[cfg(test)]
 mod tests;
+
+/// Serialization/deserialization support.
+#[path = "serialization/serialization.rs"]
```

Comment:
I have not found this to be an issue at all, but fair enough. I don't feel that strongly about it.